Gameplay.js

```diff
@@ -0,0 +1,90 @@
+class Gameplay {
```

Comment:
In the future, the "git mv" command could be used for renaming a file and keeping the version history intact.